### DISCUSSION TITLE
Welcome to blog Discussions!

### DISCUSSION THREAD

**Discussion by pycoder2000** [pycoder2000/blog#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hrishithub** [comment]
I just come across website and I've never seen such a great personal website ever! Apart from references you've provided what other resources do you refer to made this website? Any recommendations to learn how can I make one such websit? Any suggestions will be appreciated.
